js/main.js

```diff
@@ -0,0 +1,109 @@
+(function() {
+    function loadPics() {
+        var api_key, method, user_id, format, url, xmlhttp;
+
+        api_key = '74e47a159e15cbcb6139ba9c9df64c13';
+        method = "flickr.photos.search";
+        user_id = "138698049@N03";
+        format = '&format=json';
+        url = "https://api.flickr.com/services/rest"+"?method="+method+"&api_key="+api_key+"&user_id="+user_id+format+'&nojsoncallback=1';
+        xmlhttp = new XMLHttpRequest();
+
+        xmlhttp.onreadystatechange = function () {
+            var farm, server, id, title, secret, picUrl, items;
```

Comment:
Most of these variables aren't being used with `onreadystatechange`. Please move the appropriate ones down to `photoData()`